projects/observability/src/shared/dashboard/widgets/card-list/card-list-widget-renderer.component.ts

```diff
@@ -21,15 +21,23 @@ import { CardListWidgetModel } from './card-list-widget-model';
         <ht-card-list>
           <ht-card-container *ngFor="let card of cards" [ngSwitch]="this.model.cardType">
             <div class="summary-card" *ngSwitchCase="'${CardType.Summary}'">
-              <htc-summary-card [name]="card.name" [color]="card.color" [summaries]="card.summaries"></htc-summary-card>
+              <htc-summary-card
+                [name]="card.name"
+                [color]="card.color"
+                [summaries]="card.summaries"
+                (click)="this.model.clickHandler?.execute(card.context)"
+              ></htc-summary-card>
             </div>
           </ht-card-container>
         </ht-card-list>
       </htc-titled-content>
     </div>
   `
 })
-export class CardListWidgetRendererComponent<T extends Card[]> extends WidgetRenderer<CardListWidgetModel<T>, T> {
+export class CardListWidgetRendererComponent<P, T extends Card<P>[]> extends WidgetRenderer<
```

Comment:
Sorry. This typing is confusing to me.

Reply:
Looks like they are only useful in tests. Do we need to include a new generics P? What if in the card we use `unknown` instead of generic T ?

Reply:
I can do that